tests/feature-store/test_steps.py

```diff
@@ -733,6 +734,59 @@ def test_imputer_default_value(rundb_mock, engine):
     assert not imputed_df.isnull().values.any()
 
 
+class MyChoice(storey.Choice):
+    def select_outlets(self, event):
```

Comment:
just a suggestion
```suggestion 
    @staticmethod
    def select_outlets(event):
```

Reply:
`select_outlets()` is an instance method in `storey.Choice`. It's not good practice to override an instance method with a static method. It also causes static analysis warnings due to the signature not matching.